pyartcd/pyartcd/pipelines/images_health.py

```diff
@@ -38,6 +39,13 @@ async def run(self):
         if any([self.send_to_release_channel, self.send_to_forum_ocp_art]):
             await asyncio.gather(*[self._send_notifications(engine) for engine in ['brew', 'konflux']])
 
+    async def send_for_engine(self, engine):
```

Comment:
```suggestion
    def send_for_engine(self, engine):
```
Should not be async here, since we are not using `await`